docs/config.rst

```diff
@@ -125,7 +125,7 @@ General
 
    * **truncate_chars_from_path_left**: Truncates characters from the start of
      the path, showing consecutive directories as one shortened section. E.g. in
-     a directory named ``~/MyProjects/Liquidprompt/tests``, it will be shortened
+     a directory named ``~/MyProjects/Liquid Prompt/tests``, it will be shortened
```

Comment:
This isn't a explicit name reference, but a directory path. Probably should leave this one as it was, even though it is only an example.